index.html

```diff
@@ -48,12 +48,6 @@
   // editors, add as many as you like
   // only "name" is required
   editors: [{
-    name: "Orie Steele",
-    url: "https://www.linkedin.com/in/or13b/",
-    company: "Transmute",
-    companyURL: "https://www.transmute.industries/",
-    w3cid: 109171
```

Comment:
I would think you should move to formerEditors, rather than be deleted... which I can't properly suggest because that's a stanza too far for GitHub's UI.

I'm suggesting this just once here, but it probably applies to all your similar PRs.